utils/scripts/ocm/ocm.py

```diff
@@ -1166,6 +1169,54 @@ def compare_with_old_version_file(self):
         log.info("File is updated to : {} ".format(old_data))
 
 
+    def change_cluster_channel_group(self):
+        """update the channel using ocm cmd"""
+        cluster_id = self.get_osd_cluster_id()
+        run_change_channel_cmd = (
+            "ocm patch /api/clusters_mgmt/v1/clusters/{}"
+            " --body {}".format(cluster_id, self.upadte_ocm_channel_json))
+        log.info(run_change_channel_cmd)
+        ret = execute_command(run_change_channel_cmd)
+        if ret is None:
+            log.info("Failed  to Update the channel")
```

Comment:
The message can be more specific, Failed to update the channel to <channel_name>